src/static/css/nav-primary.less

```diff
@@ -0,0 +1,487 @@
+            color: @black;
+        }
+    }
+
+    .primary-nav_top-level-list > .list_item > .list_link {
+        display: block;
+        padding-bottom: @grid_gutter-width;
+        // Margin on link for larger hit area.
+        margin: 0 0 0 (@grid_gutter-width / 2);
+        font-size: unit(18px / @base-font-size-px, em);
+    }
+
+    // Hide only when js is available.
+    .sub-nav_wrapper {
+        background: @gray-5;
+        width: 100%;
+        overflow: hidden;
+        position: absolute;
+        max-height: 0;
+        left: 0;
+        z-index: 100;
+        // Maybe this will kick on the gpu?
+        -webkit-transform: translate3d(0,0,0);
+    }
+
+    // Expand the menu by looking for aria attributes.
+    // This will help us to keep our eye on a11y.
+    // The .no-js selector enables simple hover menu support for users w/o JS.
+    .sub-nav_wrapper[aria-expanded="true"],
+    .no-js .list_item:hover .sub-nav_wrapper {
+        border-top: 1px solid @gray-50;
+        border-bottom: 4px solid @gray-50;
+        z-index: 200;
+        max-height: unit(400px / @base-font-size-px, em);
+        // Maybe this will kick on the gpu?
+        -webkit-transform: translate3d(0,0,0);
+    }
+
+    // Only use transitions when allowed, JS controls transition toggle logic.
+    // Transition off.
+    .has-transition .sub-nav_wrapper {
+        transition: 0.25s max-height linear;
+    }
+    // Transition on.
+    .has-transition .sub-nav_wrapper[aria-expanded="true"] {
+        transition: 0.275s max-height ease-out;
+    }
+
+    .sub-nav_inner-wrapper {
+        padding: @grid_gutter-width 0;
+    }
+
+    .sub-nav_menu {
+        .grid_column(8);
+    }
+
+    .sub-nav_menu-list {
+        .grid_column(1, 3);
+        vertical-align: top;
+        &:first-of-type {
+            border-left-width: 0;
+        }
+    }
+
+    .sub-nav_menu-list .list_link {
+       .u-inline-block();
+        padding: unit(5px / @font-size-large, em)
+                 unit(15px / @font-size-large, em);
+        border-left-style: solid;
+        border-left-width: 5px;
+        font-size: unit(@font-size-large / @base-font-size-px, em);
+        .u-link__colors(@pacific, @pacific, @black, @black, @black,
+                        transparent, transparent, @green, @green, @green);
+        &__current {
+            .u-link__colors(@black, @black, @black, @black, @black,
+                            @green, @green, @green, @green, @green);
+            cursor: default;
+        }
+
+        .webfont-regular();
+    }
+
+    .sub-nav_menu-list .list_item {
+        display: block;
+        white-space:nowrap;
+        line-height: 2;
+    }
+
+    .sub-nav_media {
+        position: relative;
+        padding: 0 @grid_gutter-width;
+        margin-top: 0;
+        .grid_column(4);
+    }
+
+    .sub-nav_title {
+        .webfont-regular();
+        font-size: 1em;
+        color: @pacific;
+    }
+
+});
+
+.respond-to-range(801px, 814px, {
```

Comment:
It's a hackish approach anyway, if the webfonts didn't load the width of the links would differ. Will we even have one row of links in the new menu anyway? If we will and you think of a better way to determine if they've broken to a second line, go with that.